### PR TITLE
Move to f34. Update deployment notes.

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

File: coreos-ostree-importer/README.md [hunk 1]
```diff
@@ -4,6 +4,39 @@ Source code that watches for ostree-import requests on the fedora
...
+
+The application can be deployed by people with appropriate permissions by
+executing:
+
+```
+[localhost]$ ssh batcave01.iad2.fedoraproject.org
+[batcave01]$ sudo rbac-playbook openshift-apps/coreos-ostree-importer.yml
+```
+
+The Application will then be running in Fedora OpenShift instances:
```
Comment (jlebon) [inline coreos-ostree-importer/README.md:22]: Minor/optional: odd capitalization here

**Review by jlebon — APPROVED** [review]
LGTM overall; did you sanity-check e.g. the tagger and importer run fine on f34?

**Comment by dustymabe** [comment]
> LGTM overall; did you sanity-check e.g. the tagger and importer run fine on f34?

yep. They're actually already running on f34 because of https://pagure.io/fedora-infra/ansible/pull-request/571

File: coreos-ostree-importer/README.md (hunk 1 above)
Comment (dustymabe) [reply coreos-ostree-importer/README.md]: fixed

**Pull request merged**